x/wasm-storage/keeper/abci.go

```diff
@@ -109,3 +139,19 @@ func (k Keeper) ExecuteTally(ctx sdk.Context) error {
 
 	return nil
 }
+
+func tallyVMArg(inputArgs []byte, reveals map[string]RevealBody, outliers []bool) ([]string, error) {
+	argBytes, err := json.Marshal(tallyArg{
+		TallyInputArgs: inputArgs,
+		Reveals:        reveals,
+		Outliers:       outliers,
+	})
+	if err != nil {
+		return nil, err
+	}
+	arg := make([]string, 0, len(argBytes))
+	for _, b := range argBytes {
+		arg = append(arg, fmt.Sprintf("%c", b))
+	}
+	return arg, err
```

Comment:
Maybe i am wrong (To be honest i find Go a lil harder to read what exactly is going on here) but this doesn't seem like it complies with the format:

```args: ["TALLY_INPUT_ARGS", "REVEALS", "OUTLIARS"]``` or is it? if so you can ignore my comment :)

Reply:
here: [d6f112b](https://github.com/sedaprotocol/seda-chain/pull/280/commits/d6f112bf0fe59b8de70094875438195d7233d1e5)